src/main/java/microsoft/exchange/webservices/data/MessageBody.java

```diff
@@ -100,7 +100,8 @@ protected void readAttributesFromXml(EwsServiceXmlReader reader)
   @Override
   protected void readTextValueFromXml(EwsServiceXmlReader reader)
       throws XMLStreamException, ServiceXmlDeserializationException {
-    this.text = reader.readValue();
+    boolean keepWhiteSpace = this.getBodyType() == BodyType.Text;
```

Comment:
`BodyType` is an enum, so there's absolutely no need to compare using the equals method. `==` is just fine here.